sdk/communication/Azure.Communication.Administration/CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Release History
 
 ## 1.0.0-beta.2 (Unreleased)
-
+Added phone number administration. For more information, please see the [README][read_me] and [documentation][documentation].
```

Comment:
I don't think we have to add the "For more information, ..." part. That's already in the previous version changelog, and the README itself references the docs.

Reply:
I think for C# is different.
https://www.nuget.org/packages/Azure.Communication.Administration/
It seems that it picks things up from the csproj itself. But let's do that afterwards